app/models/service_template_orchestration.rb

```diff
@@ -3,6 +3,27 @@ class ServiceTemplateOrchestration < ServiceTemplate
 
   before_save :remove_invalid_resource
 
+  def self.create_catalog_item(options, _auth_user = nil)
+    transaction do
+      create(options.except(:config_info)).tap do |service_template|
+        config_info = validate_config_info(options)
+
+        service_template.orchestration_template = if config_info[:template_id]
```

Comment:
@bzwei I wanted to, but the line was too long so rubocop wasn't happy, and this was more readable than breaking that up onto two lines imo